{"name": "Search for a product in ebay home page", "status": "broken", "statusDetails": {"message": "KeyError: '62d82aa7-5675-4e3d-8007-b5df0700f16a'\n", "trace": "  File \"C:\\Python310\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python310\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_newsearch.py\", line 20, in step_impl\n    allureLogs(\"Search button is clicked\")\n  File \"features\\steps\\test_newsearch.py\", line 31, in allureLogs\n    with allure.step(text):\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\_allure.py\", line 182, in __exit__\n    plugin_manager.hook.stop_step(uuid=self.uuid, title=self.title, exc_type=exc_type, exc_val=exc_val,\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 113, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Python310\\lib\\site-packages\\allure_behave\\listener.py\", line 163, in stop_step\n    self.logger.stop_step(uuid,\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 139, in stop_step\n    self._update_item(uuid, **kwargs)\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 62, in _update_item\n    item = self._items[uuid] if uuid else self._items[next(reversed(self._items))]\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 33, in __getitem__\n    return self.thread_context.__getitem__(item)\n"}, "steps": [{"name": "Given the user is on the ebay home page", "status": "passed", "steps": [{"name": "Given the user is on the ebay home page", "status": "passed", "start": 1728969031008, "stop": 1728969031009}], "start": 1728969031008, "stop": 1728969031009}, {"name": "When the search text and make year is entered", "status": "passed", "steps": [{"name": "When the search text and make year is entered", "status": "passed", "attachments": [{"name": ".table", "source": "14e9a946-383f-44dd-a632-350b07b4c5a3-attachment.csv", "type": "text/csv"}], "start": 1728969031011, "stop": 1728969031013}], "attachments": [{"name": ".table", "source": "4a22a6c8-5755-4931-b0e8-6e6795857792-attachment.csv", "type": "text/csv"}], "start": 1728969031009, "stop": 1728969031013}, {"name": "And the search button is clicked", "status": "broken", "statusDetails": {"message": "KeyError: '62d82aa7-5675-4e3d-8007-b5df0700f16a'\n", "trace": "  File \"C:\\Python310\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python310\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_newsearch.py\", line 20, in step_impl\n    allureLogs(\"Search button is clicked\")\n  File \"features\\steps\\test_newsearch.py\", line 31, in allureLogs\n    with allure.step(text):\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\_allure.py\", line 182, in __exit__\n    plugin_manager.hook.stop_step(uuid=self.uuid, title=self.title, exc_type=exc_type, exc_val=exc_val,\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 113, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Python310\\lib\\site-packages\\allure_behave\\listener.py\", line 163, in stop_step\n    self.logger.stop_step(uuid,\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 139, in stop_step\n    self._update_item(uuid, **kwargs)\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 62, in _update_item\n    item = self._items[uuid] if uuid else self._items[next(reversed(self._items))]\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 33, in __getitem__\n    return self.thread_context.__getitem__(item)\n"}, "steps": [{"name": "And the search button is clicked", "status": "broken", "statusDetails": {"message": "KeyError: '62d82aa7-5675-4e3d-8007-b5df0700f16a'\n", "trace": "  File \"C:\\Python310\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python310\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_newsearch.py\", line 20, in step_impl\n    allureLogs(\"Search button is clicked\")\n  File \"features\\steps\\test_newsearch.py\", line 31, in allureLogs\n    with allure.step(text):\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\_allure.py\", line 182, in __exit__\n    plugin_manager.hook.stop_step(uuid=self.uuid, title=self.title, exc_type=exc_type, exc_val=exc_val,\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 113, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Python310\\lib\\site-packages\\allure_behave\\listener.py\", line 163, in stop_step\n    self.logger.stop_step(uuid,\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 139, in stop_step\n    self._update_item(uuid, **kwargs)\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 62, in _update_item\n    item = self._items[uuid] if uuid else self._items[next(reversed(self._items))]\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 33, in __getitem__\n    return self.thread_context.__getitem__(item)\n"}, "steps": [{"name": "Search button is clicked", "steps": [{"name": "Search button is clicked", "status": "passed", "start": 1728969031014, "stop": 1728969031014}], "start": 1728969031014}], "start": 1728969031013, "stop": 1728969031024}], "start": 1728969031013, "stop": 1728969031023}, {"name": "Then the search results page is displayed", "status": "skipped", "start": 1728969031024, "stop": 1728969031024}, {"name": "Then the search results page is displayed", "status": "skipped", "start": 1728969031040, "stop": 1728969031040}], "start": 1728969031008, "stop": 1728969031040, "uuid": "386ca1b7-e812-4be4-91f5-d0e8de68fdfa", "historyId": "2c87265c0b7414655403f98678c6e346", "fullName": "search functionality: Search for a product in ebay home page", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "search functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}